src/main/resources/static/css/custom.css

```diff
@@ -34,3 +34,8 @@ main {
 .pagination li.active {
 	background-color:aquamarine
 }
+
+.categoryLabel {
+	top:-1rem !important;
+	font-size: 0.8rem !important;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/matikz1110/PetFinder/file/2653733604/issues/source?bid=3239464&fileBranchId=3326233#l40)